### PR TITLE
Blockers

### PR DESCRIPTION
**Pull request by Mitchell-Conrad-20**

### CODE REVIEW

File: src/sim/UI/ViewClass.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ def __init__(self, controller):
         # Set Default States
         self.__xTextboxVal = 0
```
Comment (bhageneder) [inline src/sim/UI/ViewClass.py:21]: Should be renamed to xRobotVal and setup similar to blocker vals.

File: src/sim/UI/ViewClass.py [hunk 2]
```diff
@@ -157,6 +211,34 @@ def __yTextboxHandler(self, text):
...
+        except:
+            self.__blockerVals[0] = 0
+
+    def __yBTextboxHandler(self, text):
+        try:
+            self.__blockerVals[1] = int(text)
+        except:
+            self.__blockerVals[1] = 0
+    
+    def __widthTextboxHandler(self, text):
```
Comment (bhageneder) [inline src/sim/UI/ViewClass.py:229]: Width should only be allowed POSITIVE integers:

When I entered negative integers, I couldn't move the blocker around or enter any subsequent blockers.

File: src/sim/UI/ViewClass.py [hunk 3]
```diff
@@ -157,6 +211,34 @@ def __yTextboxHandler(self, text):
...
+        try:
+            self.__blockerVals[2] = int(text)
+        except:
+            self.__blockerVals[2] = 0
+    
+    def __heightTextboxHandler(self, text):
```
Comment (bhageneder) [inline src/sim/UI/ViewClass.py:235]: Same as Width

**Review by bhageneder — APPROVED** [review]
Look good to me and worked with a minor bug.

**Review by bhageneder — APPROVED** [review]

**Review by stoneoxmike — APPROVED** [review]
Kind of out of scope for me since I haven't worked with PyQt5 much, but looks good to my untrained eye

**Pull request merged**